sklearn/model_selection/_split.py

```diff
@@ -789,8 +825,9 @@ def split(self, X, y=None, labels=None):
             Training data, where n_samples is the number of samples
             and n_features is the number of features.
 
-        y : array-like, shape (n_samples,)
+        y : array-like, shape (n_samples,) or (n_samples, n_labels)
```

Comment:
y: array-like of length n_samples
    
Basically it can be both a multi-label or a multi-output problem. So we don't want to confuse the user